test/rulesets/Base/base.jl

```diff
@@ -216,4 +216,9 @@
         @test frule(NoRules, 1.0) === nothing
         @test rrule(NoRules, 1.0) === nothing
     end
+    
+    @testset "map" begin
+        test_rrule(map, identity, (1, 2), check_inferred=false)
```

Comment:
Tests are broken. What on earth is this trying to tell me?
```
julia> test_rrule(map, identity, (1, 2), check_inferred=false)
test_rrule: map on typeof(identity),Int64,Int64: Error During Test at /Users/me/.julia/packages/ChainRulesTestUtils/fMTso/src/testers.jl:193
  Got exception outside of a @test
  MethodError: (::ChainRulesTestUtils.var"#test_approx##kw")(::NamedTuple{(:rtol, :atol), Tuple{Float64, Float64}}, ::typeof(test_approx), ::NoTangent, ::Tangent{Tuple{Int64, Int64}, Tuple{NoTangent, NoTangent}}, ::String) is ambiguous.
  
  Candidates:
    (var"#s86"::ChainRulesTestUtils.var"#test_approx##kw")(::Any, ::typeof(test_approx), x, y::Tangent, msg)
      @ ChainRulesTestUtils ~/.julia/packages/ChainRulesTestUtils/fMTso/src/check_result.jl:124
    (var"#s85"::ChainRulesTestUtils.var"#test_approx##kw")(::Any, ::typeof(test_approx), ::AbstractZero, x, msg)
      @ ChainRulesTestUtils ~/.julia/packages/ChainRulesTestUtils/fMTso/src/check_result.jl:33
  
  Possible fix, define
    (::ChainRulesTestUtils.var"#test_approx##kw")(::Any, ::typeof(test_approx), ::AbstractZero, ::Tangent, ::Any)
  
  Stacktrace:
    [1] test_approx(actual::AbstractArray, expected::AbstractArray, msg::Any; kwargs::Base.Pairs{Symbol, V, Tuple{Vararg{Symbol, N}}, NamedTuple{names, T}} where {V, N, names, T<:Tuple{Vararg{Any, N}}})
      @ ChainRulesTestUtils ~/.julia/packages/ChainRulesTestUtils/fMTso/src/check_result.jl:80
    [2] test_approx(actual::Tuple{NoTangent, NoTangent, NoTangent}, expected::Tuple{NoTangent, NoTangent, Tangent{Tuple{Int64, Int64}, Tuple{NoTangent, NoTangent}}}, msg::Any; kwargs::Base.Pairs{Symbol, V, Tuple{Vararg{Symbol, N}}, NamedTuple{names, T}} where {V, N, names, T<:Tuple{Vararg{Any, N}}})
      @ ChainRulesTestUtils ~/.julia/packages/ChainRulesTestUtils/fMTso/src/check_result.jl:151
    [3] macro expansion
      @ ~/.julia/packages/ChainRulesTestUtils/fMTso/src/testers.jl:230 [inlined]
    [4] macro expansion
      @ ~/.julia/dev/julia/usr/share/julia/stdlib/v1.9/Test/src/Test.jl:1385 [inlined]
    [5] test_rrule(::RuleConfig, ::Any, ::Any, ::Vararg{Any}; output_tangent::Any, check_thunked_output_tangent::Any, fdm::Any, rrule_f::Any, check_inferred::Bool, fkwargs::NamedTuple, rtol::Real, atol::Real, kwargs::Base.Pairs{Symbol, V, Tuple{Vararg{Symbol, N}}, NamedTuple{names, T}} where {V, N, names, T<:Tuple{Vararg{Any, N}}})
      @ ChainRulesTestUtils ~/.julia/packages/ChainRulesTestUtils/fMTso/src/testers.jl:196
    [6] test_rrule(::Any, ::Vararg{Any}; kwargs::Base.Pairs{Symbol, V, Tuple{Vararg{Symbol, N}}, NamedTuple{names, T}} where {V, N, names, T<:Tuple{Vararg{Any, N}}})
      @ ChainRulesTestUtils ~/.julia/packages/ChainRulesTestUtils/fMTso/src/testers.jl:170
```

Reply:
```suggestion
        test_rrule(map, identity, (1.0, 2.0), check_inferred=false)
```
Should work. Basically 
```julia
julia> rand_tangent((1.0, 2.0))
Tangent{Tuple{Float64, Float64}}(3.5, 0.55)

julia> rand_tangent((1, 2))
NoTangent()
```
is the underlying issue

Reply:
Yeah, I agree: https://github.com/JuliaDiff/ChainRulesTestUtils.jl/issues/252